src/components/pagination/CdrPagination.vue

```diff
@@ -49,15 +220,15 @@
 
       <li :class="style['cdr-pagination__li--select']">
         <cdr-select
-          :id="`pagination-select-${id}`"
+          :id="`pagination-select-${uniqueId}`"
           v-model="innerValue"
           label="Navigate to page"
           hide-label
-          @change="select"
+          @change.prevent="(e) => select(e)"
```

Comment:
We can't use the `navigate` function directly here because we need to simulate a click on the corresponding page link element to in order to get some default click behavior that is not included in `navigate`